packages/quantic/force-app/main/default/lwc/quanticCitation/__tests__/quanticCitation.test.js

```diff
@@ -198,59 +226,43 @@ describe('c-quantic-citation', () => {
     });
   });
 
-  describe('when the citation source is of type Salesforce', () => {
-    it('should call the navigation mixin to get the Salesforce record URL', async () => {
-      const element = createTestComponent({
-        ...defaultOptions,
-        citation: exampleSalesforceCitation,
-      });
-      await flushPromises();
-
-      const link = element.shadowRoot.querySelector(selectors.citationLink);
-      const {pageReference} = getGenerateUrlCalledWith();
-
-      expect(pageReference.attributes.recordId).toBe(
-        exampleSalesforceCitation.fields.sfid
-      );
-      expect(link.href).toBe(`${exampleSalesforceLink}/`);
-    });
+  describe('when citation anchoring is disabled', () => {
+    describe('when the citation source is not of type Salesforce', () => {
+      it('should leave the hrefValue as the citation uri', async () => {
```

Comment:
Is that what you're trying to test?
```suggestion
      it('should add the text fragment to the hrefValue of the citation', async () => {
```